src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -420,3 +419,26 @@ def getPerformanceParameters(self, defaultParams):
         sizePerEvent = defaultParams.get('sizePerEvent', None) or 0
         memory = defaultParams.get('memoryRequirement', None) or 0
         return timePerEvent, sizePerEvent, memory
+    
+    def checkCreateFailConditoin(self, failedJob, failedReason):
+
+        valueMap = {"MaxWallTimeHours": "estimatedJobTime", 
+                    "MaxMemory": "estimatedMemoryUsage", 
```

Comment:
I'm with you Eric, not sure I really got it :-)
Brian, if
> request_disk = 1234
> +RequestDisk = 1234 
are "equivalent" (but not the same in a classad (?)). Couldn't we replace this "request_disk" by "+RequestDisk" in the WMAgent code? Would you provide the same results in the end?